apps/web/src/pages/Applications/ApplicantDashboardPage/components/DashboardHeading.tsx

```diff
@@ -39,25 +34,20 @@ import {
   isWorkExperience,
 } from "~/utils/experienceUtils";
 import { AwardExperience } from "~/api/generated";
-
-import { navigationMessages } from "@gc-digital-talent/i18n";
-import {
-  HeroCardExperienceItem,
-  HeroCardProfileItem,
-  ProfileItemStatus,
-} from "./HeroCardItem";
-import { HeroCard } from "./HeroCard";
+import { StatusItem } from "~/components/StatusItem/StatusItem";
+import { HeroCard } from "~/components/HeroCard/HeroCard";
+import { PAGE_SECTION_ID as PROFILE_PAGE_SECTION_ID } from "~/components/UserProfile/constants";
 import { PartialUser } from "../types";
 
-function deriveSectionStatus(
-  isMissingRequiredFields: boolean,
-  isMissingOptionalFields: boolean,
-): ProfileItemStatus {
-  if (isMissingRequiredFields) return "has-empty-required-fields";
-  if (isMissingOptionalFields) return "has-empty-optional-fields";
-  return "all-sections-complete";
+function buildLink(href: string, chunks: React.ReactNode): React.ReactElement {
+  return <Link href={href}>{chunks}</Link>;
```

Comment:
I'm sorry! When you merged the link update it switched the default to `secondary` which doesn't stand out enough. We will need to set the colour for this 😬

Reply:
Hah, I saw that this morning and thought it was just a visited link.  I didn't realize it was linked to the button/link refresh and opened #6899.  I just reset it back to white like the ScrollToLink.
Ref: [Fix subtitle link color](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6823/commits/a8d129bf371066003011e540c1b3eb4fcb416323)